lib/service_manager.dart

```diff
@@ -51,6 +52,30 @@ class ServiceConnectionManager {
 
   Stream<Null> get onConnectionClosed => _connectionClosedController.stream;
 
+  Future<Response> callService(String name, {Map args}) {
```

Comment:
Add a comment describing the restriction that there must be only one registered provider of the service.

Reply:
done.